contracts/ethregistrar/LinearPremiumPriceOracle.sol

```diff
@@ -6,35 +6,45 @@ import "./StablePriceOracle.sol";
 contract LinearPremiumPriceOracle is StablePriceOracle {
     using SafeMath for *;
 
-    uint GRACE_PERIOD = 90 days;
+    uint256 GRACE_PERIOD = 90 days;
 
-    uint public initialPremium;
-    uint public premiumDecreaseRate;
+    uint256 public immutable initialPremium;
+    uint256 public immutable premiumDecreaseRate;
 
-    bytes4 constant private TIME_UNTIL_PREMIUM_ID = bytes4(keccak256("timeUntilPremium(uint,uint"));
+    bytes4 private constant TIME_UNTIL_PREMIUM_ID =
+        bytes4(keccak256("timeUntilPremium(uint,uint"));
 
-    constructor(AggregatorInterface _usdOracle, uint[] memory _rentPrices, uint _initialPremium, uint _premiumDecreaseRate) public
-        StablePriceOracle(_usdOracle, _rentPrices)
-    {
+    constructor(
+        AggregatorInterface _usdOracle,
+        uint256[] memory _rentPrices,
+        uint256 _initialPremium,
+        uint256 _premiumDecreaseRate
+    ) public StablePriceOracle(_usdOracle, _rentPrices) {
         initialPremium = _initialPremium;
         premiumDecreaseRate = _premiumDecreaseRate;
     }
 
-    function _premium(string memory name, uint expires, uint /*duration*/) override internal view returns(uint) {
+    function _premium(
+        string memory name,
+        uint256 expires,
+        uint256 /*duration*/
+    ) internal view override returns (uint256) {
         expires = expires.add(GRACE_PERIOD);
-        if(expires > block.timestamp) {
+        if (expires > block.timestamp) {
             // No premium for renewals
             return 0;
         }
 
         // Calculate the discount off the maximum premium
-        uint discount = premiumDecreaseRate.mul(block.timestamp.sub(expires));
```

Comment:
In 0.8, we can replace all safemath stuff with regular arithmetic, because safe math is the default now.